WordPress/src/main/res/values/strings.xml

```diff
@@ -1497,6 +1497,15 @@
     <string name="notifications_tab_title_follows">Follows</string>
     <string name="notifications_tab_title_likes">Likes</string>
 
+    <!-- Notification permission -->
+    <string name="notifications_permission_fix">Fix</string>
+    <string name="notifications_permission_dismiss_content_description">Dismiss notification permission warning.</string>
+    <string name="notifications_permission_off_desc">Push notifications are turned off.</string>
+    <string name="notifications_permission_bottom_sheet_title">Push notifications are turned off</string>
+    <string name="notifications_permission_bottom_sheet_description_1">You’ll need to open the app to see notifications.</string>
```

Comment:
Missing escape \ character before ' in You'll

Reply:
I've learned that `’` and `'` are different. 😄 I copied strings from Figma, which uses `’`, that's why the IDEA didn't show an error. But I've changed `’` to `'`. They both are okay but standard `'` is more popular in our strings.

Done: 6ee77da96b346ea0569b234c9d1f21c96168c19d